server/validators/decorators.ts

```diff
@@ -0,0 +1,15 @@
+import { registerDecorator, ValidationOptions } from 'class-validator'
```

Comment:
Field decorator which takes in another SimpleTime object and compares it. Will pass if the field being validated is a time after the field its being compared against

```
earlierTime: SimpleTime

@IsSimpleTimeAfter('earlierTime', { message: 'nowTime should be after the earlierTime' })
nowTime: SimpleTime 
```

Reply:
*facepalm* spent time doing this then realised we probably NEED to be able to put in for example 7pm in the evening curfewStart and 7am in the morning curfew end _**the next day**_

Oh well, I'm going to leave the decorator in for now in case it comes in useful or so that we have a reference on how to do custom decorators later on

Reply:
Agree - leave it in as the reference example.